src/containers/Network/Nodes.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import axios from 'axios'
+import { useTranslation } from 'react-i18next'
+import { useQuery } from 'react-query'
+import NetworkTabs from './NetworkTabs'
+import Map from './Map'
+import NodesTable from './NodesTable'
+import Log from '../shared/log'
+import { FETCH_INTERVAL_NODES_MILLIS, localizeNumber } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+
+export const Nodes = () => {
+  const language = useLanguage()
+  const { t } = useTranslation()
+  const [nodes, setNodes] = useState<any>([{}])
+  const [locations, setLocations] = useState([])
+  const [unmapped, setUnmapped] = useState(0)
+
+  useQuery('fetchNodesData', async () => fetchData(), {
```

Comment:
You can have nodes come from this and return it in fetchData.  That will remove an extra useState and setNodes.